v20.1/cockroach-commands.md

```diff
@@ -30,6 +30,7 @@ Command | Usage
 [`cockroach debug zip`](cockroach-debug-zip.html) | Generate a `.zip` file that can help Cockroach Labs troubleshoot issues with your cluster.
 [`cockroach debug merge-logs`](cockroach-debug-merge-logs.html) | Merge multiple log files from different machines into a single stream.
 [`cockroach workload`](cockroach-workload.html) | Run a built-in load generator against a cluster.
+[`cockroach nodelocal upload`](cockroach-nodelocal-upload.html) | <span class="version-tag">New in v20.1:</span> Upload a file to a node's local file system.
```

Comment:
maybe "Upload a file *to the external-IO-dir on* a node's...". Just want to get out ahead of the security concerns this feature might immediately raise if someone didn't know that restriction was in place.